src/main/java/hockeyapp/HockeyappApplication.java

```diff
@@ -28,11 +29,14 @@
 public class HockeyappApplication implements Describable<HockeyappApplication> {
     public static final long SCHEMA_VERSION_NUMBER = 1L;
 
+    @SuppressFBWarnings(value = {"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"}, justification = "Breaks binary compatibility if removed.")
     @XStreamAsAttribute
-    public long schemaVersion;
+    public long schemaVersion; // TODO: Fix Findbugs gracefully.
```

Comment:
Should it be marked as deprecated then? Just in case thee is really no usages anywhere (including Jelly and Co)